stacks/openebs-nfs-provisioner/deploy.sh

```diff
@@ -5,15 +5,14 @@ set -e
 ################################################################################
 # repo
 ################################################################################
-helm repo add openebs-nfs https://openebs.github.io/dynamic-nfs-provisioner
+helm repo add openebs https://openebs.github.io/openebs
 helm repo update > /dev/null
 
 ################################################################################
 # chart
 ################################################################################
 STACK="openebs-nfs-provisioner"
 CHART="openebs-nfs/nfs-provisioner"
-CHART_VERSION="0.11.0"
```

Comment:
Do we need a chart version? or is it always going to install the latest one?

Reply:
yes, last one